server/plugin/mm_34646_token_refresh.go

```diff
@@ -63,14 +80,16 @@ func (p *Plugin) forceResetAllMM34646() error {
 			time.Sleep(delayBetweenUsers)
 		}
 
-		if len(keys) < pageSize {
+		if len(keys) == 0 {
 			break
 		}
 	}
+
+	_ = p.API.KVSet(mm34646DoneKey, []byte("done"))
```

Comment:
@levb - could we have used instead of "done" `tryInfo.MM34646ResetTokenDone`?

Reply:
sure, but the key name is already a reference